homeassistant/components/blinkt/light.py

```diff
@@ -72,25 +71,10 @@ def hs_color(self):
         return self._hs_color
 
     @property
-    def supported_features(self):
-        """Flag supported features."""
-        return SUPPORT_BLINKT
-
-    @property
-    def is_on(self):
+    def is_on(self) -> bool:
         """Return true if light is on."""
         return self._is_on
```

Comment:
Can be removed when `self._attr_is_on` is used throughout the rest of the code (L93, L100). See comment above.

Reply:
I thought this had to be left in so the state can update.

Reply:
Don't we have to leave this property in so the state can change?

Reply:
That is inherited from ToggleEntity.